src/components/CubeDraft.js

```diff
@@ -98,26 +98,31 @@ const CubeDraft = ({ draft, socket }) => {
     let status = 'inProgress';
     while (status === 'inProgress') {
       // we want to do this first as we don't want to spam the server
-      await new Promise((resolve) => setTimeout(resolve, 2000));
+      await new Promise((resolve) => {
+        setTimeout(resolve, 2000);
+      });
 
       try {
         const res = await callApi('/multiplayer/trybotpicks', {
           draft: draft.id,
         });
         if (res) {
-          let json = await res.json();
+          const json = await res.json();
           status = json.result;
 
           console.log(json);
 
-          if (json.picks == 0) {
-            await new Promise((resolve) => setTimeout(resolve, 5000));
+          if (json.picks === 0) {
+            await new Promise((resolve) => {
+              setTimeout(resolve, 5000);
+            });
+            // eslint-disable-next-line no-continue
```

Comment:
This feels like another rule that is overly opinionated and could be globally disabled.

Reply:
Generally, this rule encourages you to refactor long bodies in loops into smaller functions. A return from the extracted function can serve the same purpose as a continue inside the loop.

I think this [stackoverflow post](https://softwareengineering.stackexchange.com/questions/434124/are-there-any-problems-with-using-continue-or-break) explains the reasoning well. A continue isn't always bad, but its good to consider other tools in the toolbox.

Are there are lot of other continue statements elsewhere causing lint noise too?